homeassistant/components/flexpool/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Config flow for Flexpool integration."""
+import logging
+import re
+
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+
+from ...const import CONF_ADDRESS
```

Comment:
Only use relative imports within the same package.

```suggestion
from homeassistant.const import CONF_ADDRESS
```